x-pack/test/apm_api_integration/tests/service_nodes/get_service_nodes.spec.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import archives from '../../common/fixtures/es_archiver/archives_metadata';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+  const supertest = getService('legacySupertestAsApmReadUser');
```

Comment:
We're in the process of moving to `apmApiClient`:

```suggestion
  const apmApiClient = getService('apmApiClient');
```